sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_models.py

```diff
@@ -1281,24 +1281,34 @@ class JobMatchingMode:
     :ivar suspend_mode: Any object.
     :vartype suspend_mode: JSON
     """
+    @overload
+    def __init__(
+        self,
+        *
+        queue_and_match_mode: JSON,
+        model_type: Literal["queueAndMatchMode"] = JobMatchModeType.QUEUE_AND_MATCH_MODE
+    ) -> None:
+        ...
 
-    _validation = {}
-
-    _attribute_map = {
```

Comment:
None of your models should need these private dictionary attributes. They were internal to the legacy models.